worlds/openrct2/Items.py

```diff
@@ -0,0 +1,103 @@
+from BaseClasses import Item
+import typing
+import random
+import copy
+from .Constants import *
+from .Options import *
+
+
+class OpenRCT2Item(Item):
+    game: str = "OpenRCT2"
+
+
+
+# def set_openRCT2_items(scenario, rules, monopoly_mode, include_gamespeed_items, furry_convention_traps, spam_traps, bathroom_traps, skips, filler, atm, first_aid):
+def set_openRCT2_items(world):
+    # print("\nThis is the selected scenario:")
+    # print(scenario)
+    # print("And these items will be randomized:")
+    # print(Scenario_Items[scenario])
+    openRCT2_items = copy.deepcopy(Scenario_Items[world.options.scenario.value])
+    rules = [world.options.difficult_guest_generation.value,
+                 world.options.difficult_park_rating.value,
+                 world.options.forbid_high_construction.value,
+                 world.options.forbid_landscape_changes.value,
+                 world.options.forbid_marketing_campaigns.value,
+                 world.options.forbid_tree_removal.value]
+
+    if world.options.include_atm:
+        if "Cash Machine" not in openRCT2_items:
+            openRCT2_items.append("Cash Machine")
+
+    if world.options.include_first_aid:
+        if "First Aid" not in openRCT2_items:
+            openRCT2_items.append("First Aid")
+
+    if world.options.monopoly_mode.value:
+        count = 0
+        while count < 20:
+            openRCT2_items.append("Land Discount")
+            openRCT2_items.append("Construction Rights Discount")
+            count += 1
+            
+    if world.options.include_gamespeed_items.value:
+        count = 0
+        while count < 4:
+            openRCT2_items.append("Progressive Speed")
+            count += 1
+
+    count = 0
+    while count < world.options.furry_convention_traps.value:
+        openRCT2_items.append("Furry Convention Trap")
+        count += 1
+    count = 0
+    while count < world.options.spam_traps.value:
+        openRCT2_items.append("Spam Trap")
+        count +=1
+    count = 0
+    while count < world.options.bathroom_traps.value:
+        openRCT2_items.append("Bathroom Trap")
+        count +=1
+    count = 0
+    while count < world.options.skips.value:
+        openRCT2_items.append("Skip")
+        count +=1
+
+    for number, rule in enumerate(item_info["park_rules"]):#Check every rule type
+        if rules[number] == 1:#If it's enabled and can be disabled
+            openRCT2_items.append(rule)#Add an item to disable
```

Comment:
```suggestion
    for number, rule in enumerate(item_info["park_rules"]):  # Check every rule type
        if rules[number] == 1:  # If it's enabled and can be disabled
            openRCT2_items.append(rule)  # Add an item to disable
```

Feel free to ignore this one if you don't like it like this